### PR TITLE
print startup time in version upload

### PR DESCRIPTION
**Pull request by taylorlee**
## What this PR solves / how to test

Fixes version upload missing from version upload. Similar to https://github.com/cloudflare/workers-sdk/pull/6318 
Removes bundle-size warning.

## Author has addressed the following
- [x] Wait for API release with field

- Tests
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [x] I don't know
  - [ ] Required / Maybe required
  - [ ] Not required because:
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [x] Not necessary because: already documented at https://developers.cloudflare.com/workers/platform/limits/#worker-startup-time 
  API docs at api.cloudflare.com already have startup_time_ms field added

### CODE REVIEW

File: .changeset/moody-baboons-wonder.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+log startup time in version upload
```
Comment (petebacondarwin) [inline .changeset/moody-baboons-wonder.md:5]:
```suggestion
feat: log Worker startup time in the `version upload` command
```

See https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changeset-message-format

File: .changeset/witty-bats-drum.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
```
Comment (petebacondarwin) [inline .changeset/witty-bats-drum.md:2]:
```suggestion
"wrangler": patch
```

File: .changeset/witty-bats-drum.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+remove bundle size warning
```
Comment (petebacondarwin) [inline .changeset/witty-bats-drum.md:5]: fix: remove bundle size warning from Worker deploy commands

Bundle size was a proxy for startup time. Now that we have startup time
reported, focus on bundle size is less relevant.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
A couple of fixups on the changesets.
Marked as blocked till the backend API is in place.

**Comment by petebacondarwin** [comment]
There are some e2e tests that need their snapshots updating: See https://github.com/cloudflare/workers-sdk/actions/runs/10176257471/job/28145228623?pr=6385

**Comment by petebacondarwin** [comment]
Also there are some formatting issues, which can be addressed by running `pnpm fix` in the root.

**Comment by taylorlee** [comment]
moving to #6392 for CI

**Comment by taylorlee** [comment]
> There are some e2e tests that need their snapshots updating: See https://github.com/cloudflare/workers-sdk/actions/runs/10176257471/job/28145228623?pr=6385

I think the snapshots were updated, I believe the run failed as expected due to API fields not yet being released.

**Pull request closed without merging**